src/bin/tikv-server.rs

```diff
@@ -276,6 +277,19 @@ fn get_rocksdb_db_option(config: &toml::Value) -> RocksdbOptions {
     let wal_recovery_mode = util::config::parse_rocksdb_wal_recovery_mode(rmode).unwrap();
     opts.set_wal_recovery_mode(wal_recovery_mode);
 
+    let wal_dir = get_toml_string(config, "rocksdb.wal-dir", Some("".to_owned()));
+    if !wal_dir.is_empty() {
+        opts.set_wal_dir(&wal_dir)
+    };
+
+    let wal_ttl_seconds = get_toml_int(config, "rocksdb.wal-ttl-seconds", Some(0));
```

Comment:
I'll rename the option to wal-ttl instead of wal-ttl-seconds.
What should I do with values without unit ?
Do you prefer I do nothing (will be ms like other config values) or I should consider to accept default values in seconds for this config ?